### ISSUE TITLE
[Feature Request] Support fallback font

### ISSUE THREAD

**Issue by pexcn** [mintty/mintty#883]
I changed mintty's font settings to `consolas` (system language: Simplified Chinese), but the `consolas` font only supports English characters. So windows will auto fallback other characters to system default fonts, other characters will look very ugly (at least in Simplified Chinese), like this:
![image_2019-06-10_23-19-56](https://user-images.githubusercontent.com/4590439/59206113-d4be1500-8bd6-11e9-81dd-a282a0d4baae.png)

So I hope that mintty will support fallback font settings in future versions. Just like the font settings of the Intellj IDEA:
![image_2019-06-10_23-17-17](https://user-images.githubusercontent.com/4590439/59205725-0b476000-8bd6-11e9-8653-55efde2123fa.png)

**Comment by BrianInglis** [comment]
mintty already uses Uniscribe for output, so will use Windows linked fonts if specified in the registry: 
https://docs.microsoft.com/en-us/globalization/input/font-technology#font-linking 
which you have to set up yourself for standard fonts. 
Windows "font fallback" is the original facility to use Windows equivalent fonts names as a substitute for font names available on other platforms, e.g. Times->Times New Roman, Courier->Courier New, Swiss->Arial, etc.

**Comment by pexcn** [comment]
Thanks for your reply, I will try it later.

**Comment by pexcn** [comment]
Solved, thank you.

**Comment by mintty** [comment]
Mintty now supports script-specific secondary fonts, option `FontChoice`. Configuration example:
```
FontChoice=Hebrew:6;Arabic:7;Han:8;Hangul:9
Font6=David
Font7=Simplified Arabic Fixed
Font8=FangSong
Font9=MingLiU
```

**Comment by mintty** [comment]
Released 3.0.2.
